packages/ra-core/src/controller/list/useListParams.ts

```diff
@@ -88,15 +88,23 @@ export const useListParams = ({
     const location = useLocation();
     const navigate = useNavigate();
     const [localParams, setLocalParams] = useState(defaultParams);
-    const [params, setParams] = useStore(storeKey, defaultParams);
+    const [params, setParams] = useStore(
+        storeKey || `${resource}.listParams`,
```

Comment:
if `storeKey` is false, this will still use the store. Can you add a comment to explain what's the rationale?

Reply:
We can't disable a hook, so although we still read the store, its values are not used later in the code